modules/ROOT/pages/kubernetes/operations/backup-restore.adoc

```diff
@@ -101,11 +102,11 @@ For more information about the available backup parameters, see <<kubernetes-neo
 ----
 neo4j:
   image: "neo4j/helm-charts-backup"
-  imageTag: "4.4.20"
+  imageTag: "4.4.22"
   jobSchedule: "* * * * *"
   successfulJobsHistoryLimit: 3
   failedJobsHistoryLimit: 1
-  backoffLimit: 1
+  backoffLimit: 3
```

Comment:
below this please add multiple databases...replace neo4j with neo4j,system

Reply:
aah, good catch!